systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -1634,6 +1664,32 @@ void testChangingInternalToExternalLoggingTriggerRollingUpdate() {
         RollingUpdateUtils.waitTillComponentHasRolledAndPodsReady(testStorage.getNamespaceName(), testStorage.getBrokerSelector(), 3, brokerPods);
     }
 
+    /**
+     * Waits for the propagation of log level changes within the Kafka Connect configuration and verifies the change across all specified Connect pods.
+     * This method first checks if the log level change is reflected in the Kafka Connect REST API's logger settings. Then, it validates
+     * that the new log level is being applied in the actual log outputs in last 60 seconds.
+     *
+     * @param testStorage      The {@link TestStorage} contains test details.
+     * @param connectPods      A map of Kafka Connect Pods.
+     * @param scraperPodName   The name of the pod used to perform network requests within the cluster.
+     * @param connectLogMatch  A {@link Predicate<String>} that tests log entries to confirm the presence of the new log level.
+     * @param logLevel         The desired log level (e.g., "INFO", "DEBUG") that the method waits to be propagated to the Kafka Connect
+     *                         configuration and reflected in the logs.
+     */
+    private static void waitForConnectLogLevelChangePropagation(TestStorage testStorage, Map<String, String> connectPods, String scraperPodName, Predicate<String> connectLogMatch, String logLevel) {
+        LOGGER.info("Waiting for log4j.properties will contain desired settings");
+        TestUtils.waitFor("Logger change", TestConstants.GLOBAL_POLL_INTERVAL, TestConstants.GLOBAL_TIMEOUT,
+            () -> cmdKubeClient().namespace(testStorage.getNamespaceName()).execInPod(scraperPodName, "curl", "http://" + KafkaConnectResources.serviceName(testStorage.getClusterName())
+                + ":8083/admin/loggers/root").out().contains(logLevel)
+        );
+
+        TestUtils.waitFor(String.format("wait till change in Log level %s takes place in actual Pods", logLevel), Duration.ofMillis(100).toMillis(), TestConstants.SAFETY_RECONCILIATION_INTERVAL,
```

Comment:
Do you need to execute 10 times this command in one second?

Reply:
good point thanks.